assignment7/allClients.js

```diff
@@ -0,0 +1,18 @@
+var express = require('express');
+var router = express.Router();
+var rp = require('request-promise')
```

Comment:
@abansal26 What is rp here? Why you're repeating same mistake again?

Reply:
I used rp as abbrevation but i changed it as suggested. Commit ID - 2b780672efb014cb63fc8cff825b998ae65d0206